### ISSUE TITLE
Tag is being read few times in a row

### ISSUE THREAD

**Issue by andrewyazura** [brobots-hub/meal-tracking#2]
When I try to read a tag, the `card_inserted` is triggered 3 or more times.

**Comment by andrewyazura** [comment]
Also I noticed it's really slow and can display that tag is read after a minute or so. Feels like something blocks it, then releases and it shows all tags it has read in that period

**Comment by andrewyazura** [comment]
I feel like I'll have to rewrite library. I noticed that some things could be changed to improve it. Hopefully, there won't be any need to touch library's code

**Comment by andrewyazura** [comment]
Found out that "easy mode" code presented on `rdm3600` library page works fine:
```python
import rdm6300

reader = rdm6300.Reader('/dev/ttyS0')
print("Please insert an RFID card")
while True:
    card = reader.read()
    if card:
        print(f"[{card.value}] read card {card}")
```

While the "real-life scenario" option is bugged and delayed. Temporarily I'll use the code shown above to read tags, but maybe I'll have to modify the library itself in the future

**Comment by andrewyazura** [comment]
Even after making these changes, the issue remains. It could be that my code is somehow overlapping with the reader

**Comment by andrewyazura** [comment]
As I thought, the library is fine. I disabled the callback in `Reader` class and it works as it has to.

**Comment by andrewyazura** [comment]
I moved the request to `card_removed` and it's responsive now, with no lags. The only issue is now it sometimes prints `WARNING:root:[/dev/serial0] got trash resetting RFID read to assume we are at the beginning`. I checked the values returned by the RFID reader and there were `b'\xf8'`. I don't know what can cause this behavior, but this is a topic for a different issue

**Comment by andrewyazura** [comment]
Created a new issue for trash values - #3

**Comment by andrewyazura** [comment]
Closed with #4
